packages/core/content-manager/server/src/history/index.ts

```diff
@@ -20,6 +20,9 @@ const getFeature = (): Partial<Plugin.LoadedPlugin> => {
         // Start recording history and saving history versions
         getService(strapi, 'history').init();
       },
+      destroy({ strapi }) {
```

Comment:
destroy matches the Strapi lifecycle name, but init doesn't. It would be better if it was consistent, so probably bootstrap and destroy as method names